src/translations/dynamic/csv-locale-en.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const csv_locale_en = {
+    // Used in the tabular creation and modification
+    'TabularCreationSkeletonComment/GENERATOR':
```

Comment:
'.' instead of '/' as separator in all keyx

Reply:
Done